packages/app-mobile/components/screens/move-notebook.js

```diff
@@ -0,0 +1,231 @@
+        return <View style={{
+            marginTop: 15,
+            marginBottom: 15,
+            flex: -1,
+            borderBottomWidth: 1,
+            borderBottomColor: theme.dividerColor
+        }} key={key}></View>;
+    }
+
+    render() {
+        let items = [];
+
+        const theme = themeStyle(this.props.themeId);
+
+        // HACK: inner height of ScrollView doesn't appear to be calculated correctly when
+        // using padding. So instead creating blank elements for padding bottom and top.
+        items.push(<View style={{height: theme.marginTop}} key="bottom_top_hack"/>);
+
+        items.push(
+            <View key={'folder_header'} style={{flex: 1, flexDirection: 'row', alignSelf: 'center'}}>
+                <View key={'folder_header'}>
+                    <Text style={this.styles().sideButtonTextBold}>{'Choose Notebook'}</Text>
+                </View>
+            </View>
+        );
+
+        items.push(this.makeDivider('divider_all'));
+
+        items.push(
+            <View key={'root'} style={{flex: 1, flexDirection: 'row'}}>
+                <Icon name={'folder-open-outline'} style={this.styles().sidebarIcon}/>
+                <TouchableOpacity key={'root'} onPress={() => this.rootfolder_press()}>
+                    <View key={'root'}>
+                        <Text style={this.styles().sideButtonText}>{'Root Folder'}</Text>
+                    </View>
+                </TouchableOpacity>
+            </View>
+        );
+
+        if (this.props.folders.length) {
+            const result = shared.renderFolders(this.props, this.renderFolderItem, false);
+            const folderItems = result.items;
+            items = items.concat(folderItems);
+        }
+
+        const style = {
+            flex: 1,
+            borderRightWidth: 1,
+            borderRightColor: theme.dividerColor,
+            backgroundColor: theme.backgroundColor,
+        };
+
+        return (
+            <View style={style}>
+                <View style={{flex: 1, opacity: this.props.opacity}}>
+                    <ScrollView scrollsToTop={false} style={this.styles().menu}>
+                        {items}
+                    </ScrollView>
+                </View>
+                <dialogs.DialogBox
+                    ref={dialogbox => {
+                        this.dialogbox = dialogbox;
+                    }}
+                />
+            </View>
+        );
+    }
+}
+
+const MoveNotebookScreen = connect(state => {
+    return {
+        folders: state.folders,
+        selectedFolderId: state.selectedFolderId,
+        notesParentType: state.notesParentType,
+        themeId: state.settings.theme,
+        // Don't do the opacity animation as it means re-rendering the list multiple times
+        // opacity: state.sideMenuOpenPercent,
+        collapsedFolderIds: state.collapsedFolderIds,
+    };
+})(MoveNotebookScreenComponent);
+
+module.exports = {MoveNotebookScreen};
```

Comment:
This is copy and pasted from the sidebar code. I'm afraid we can't have such copy and paste in the codebase as we'll soon be unable to maintain it.

I'm going to close the pull request for now as it won't be merged as it is - but I suggest you propose how you'd like to implement this in the original GitHub issue, possibly with screenshots, so that we can agree on the implementation.

Reply:
the conversation in that issue is locked ..

Reply:
It's reopened now.

Reply:
It's saying This conversation has been locked and limited to collaborators.

Reply:
Right, I didn't notice it was locked. It's unlocked now.